app/ledger.js

```diff
@@ -1692,6 +1714,50 @@ var callback = (err, result, delayTime) => {
   run(delayTime)
 }
 
+var retrieveCustomerTransactions = (params, callback) => {
+  var options = {
+    url: process.env.ADDFUNDS_URL + '/v1/refunds/' + params.address,
+    method: params.method || 'GET',
+    responseType: 'text',
+    headers: underscore.defaults(params.headers || {}, { 'content-type': 'application/json; charset=utf-8' }),
+    verboseP: params.verboseP || false
+  }
+  request.request(options, (err, response, body) => {
+    if (err) {
+      return callback(null)
+    }
+    var transactions = response.statusCode === 200 && body ? JSON.parse(body) : null
+    if (transactions && clientOptions.debugP) {
+      console.log(transactions)
+    }
+    callback(transactions)
+  })
+}
+
+var retrieveCustomerInfo = (params, callback) => {
+  var options = {
+    url: process.env.ADDFUNDS_URL + '/v1/customers/' + params.address,
+    method: params.method || 'GET',
+    responseType: 'text',
+    headers: underscore.defaults(params.headers || {}, { 'content-type': 'application/json; charset=utf-8' }),
+    verboseP: params.verboseP || false
+  }
+  request.request(options, (err, response, body) => {
+    if (err) {
+      return callback(null)
+    }
+    var customer = response.statusCode === 200 ? JSON.parse(body) : null
+    if (customer && process.env.LEDGER_LAST_PURCHASE_DAYS_AGO) {
+      customer.lastChargeTimestamp = parseInt(moment().subtract(parseInt(process.env.LEDGER_LAST_PURCHASE_DAYS_AGO), 'days').format('x'))
+    }
+    if (customer && clientOptions.debugP) {
```

Comment:
Should we consider logging non-200 responses? ex: If we're constantly getting an error, would be nice to know if it's a 500 error (server) or a 401 (access)... along with details about the error 😄  (presumably in body)